mkosi/__init__.py

```diff
@@ -6323,12 +6330,15 @@ def load_args(args: argparse.Namespace) -> CommandLineArguments:
     if args.ssh and not args.base_packages:
         die("--ssh cannot be installed without --base-packages")
 
-    if args.ssh and not args.network_veth:
-        die("--ssh cannot be used without --network-veth")
+    if args.ssh and not args.network_veth and args.ssh_port == 22:
+        die("--ssh cannot be used without --network-veth or a custom --ssh-port")
```

Comment:
This assumes you have ssh running locally… But if it's not, that should be OK, no?